README.md

```diff
@@ -1,47 +1,181 @@
-# node-neutralino
+# Node Neutralino
 
-Develop Neutralinojs apps with a Node.js backend
+[<img src="https://github.com/user-attachments/assets/379553c3-bf97-43c9-8a7f-b338e0cdd9cf" align="right" width="100">](https://github.com/user-attachments/assets/379553c3-bf97-43c9-8a7f-b338e0cdd9cf)
 
-### Goal
+Node Neutralino is a NPM package that lets you write backend code for your [Neutralinojs](https://neutralino.js.org) Apps, It supports Vanilla JS as well as Frontend Frameworks.
 
-```js
-const { NeutralinoApp } = require("@neutralinojs-community/node-neu");
+---
+
+You could manually add Node Neutralino to your projects or Get started with one of the premade templates!
+
+Templates:
+- [Vanilla JS](https://github.com/neutralinojs-community/node-neutralino-vanilla)
+- [Vite + React.js + TS](https://github.com/neutralinojs-community/node-neutralino-react)
+
+### Example Usage:
 
-const app = new NeutralinoApp({
-  url: "/resources",
-  modes: {
-    window: {
-      width: 500,
-      height: 500,
-    },
-  },
-});
+- #### Using Templates
 
-app.events.on("testEvent", () => {
-  // Node.js code
-});
+```bash
+# Use neutralinojs-community/node-neutralino-vanilla for Vanilla JS template
+$ neu create myapp --template neutralinojs-community/node-neutralino-react
+$ cd myapp
+
+# Run Neu App
+$ neu run
+
+# Build Neu App
+$ neu build --clean
 ```
----
+- #### Manual Instructions
+
+> - Add `node-neutralino` as npm dependency in root of your NEU App.
+> ```bash
+> $ npm i node-neutralino
+> ```
+> 
+> - Add config in `neutralino.config.json` for projectRunner
+> ```json
+> // This is required since node-neutralino communicates via neutralinojs extension protocol.
```

Comment:
Could you please use `js` for code highlighting. Now, GitHub renders some lines in red since they are not valid JSON statements. Thanks